wpeverest/everest-forms
# Feature - Bootstraping basic PHPUnit routines.

### All Submissions:






### Changes proposed in this Pull Request:
 
Feature - PHPUnit test routines

* 8253e17 Semantics in phpunit directories

* 276016a Feat - now can accurately assert all major filters for EVF

* 354e27c Feature - Tests now for action hooks as well

* d965026 Function name semantics

* e60d802 Feat - adding db table and consistency checks

* e4be682 Feature - added check for empty log file

* b30062c Finalizing + Docblocks

* 7616d56 Feat - Add phpunit to installed deps

If error is encountered:
![image](https://user-images.githubusercontent.com/30156167/78994174-29b41a00-7b5f-11ea-9818-1d38fe16bc71.png)


If no error is encountered:
![image](https://user-images.githubusercontent.com/30156167/78994197-30db2800-7b5f-11ea-80d6-c3b007965d50.png)



Closes # .

### How to test the changes in this Pull Request:

1. Use composer install.
2. Run the test generation script under `bin` with test parameters. See 3 for better idea.
3. `./bin/install-wp-tests.sh <db-name> <db-user> <db-pass> <db-host>`
4. `./vendor/bin/phpunit`.
5. Observe that test commences.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Bootstraping basic PHPUnit routines.
